src/Network/Connection.php

```diff
@@ -387,7 +421,7 @@ private function writeData(Frame $stompFrame, $timeout)
         $size = strlen($data);
         $lastByteTime = microtime(true);
         do {
-            $written = @fwrite($this->connection, substr($data, $offset), 81920);
```

Comment:
why did we increase this buffer in this past?

Reply:
We really had reduced it, previously we trying to write all the message length at once. But still it seems that it was not enough.